docs/core/docker/intro-net-docker.md

```diff
@@ -1,191 +1,66 @@
-
-* [ASP.NET Framework images on DockerHub](https://hub.docker.com/r/microsoft/aspnet/)
-
-* [ASP.NET Web Forms app on .NET Framework 4.6.2 sample](https://github.com/Microsoft/dotnet-framework-docker-samples/tree/master/aspnetapp)
-
-#### Windows Communication Framework (WCF)
-
-* [Windows Communication Framework (WCF) images on DockerHub](https://hub.docker.com/r/microsoft/wcf/)
-
-* [Windows Communication Framework (WCF) images on GitHub](https://github.com/microsoft/wcf-docker)
-
-* [Windows Communication Framework (WCF) Docker samples using .NET Framework 4.6.2](https://github.com/Microsoft/wcf-docker-samples)
-
-#### Internet Information Server (IIS)
-
-* [Internet Information Server (IIS) images on DockerHub](https://hub.docker.com/r/microsoft/iis/)
-
-* [Internet Information Server (IIS) images on GitHub](https://github.com/microsoft/iis-docker)
-
-### Interact with other Microsoft stack container images
-
-#### Microsoft SQL Server
-
-* [Run the Microsoft SQL Server for Linux 2017 container image with Docker Quickstart](https://docs.microsoft.com/sql/linux/quickstart-install-connect-docker)
-
-* [Microsoft SQL Server for Linux images on DockerHub](https://hub.docker.com/r/microsoft/mssql-server-linux/)
-
-* [Microsoft SQL Server Express Edition images for Windows Containers on DockerHub](https://hub.docker.com/r/microsoft/mssql-server-windows-express/)
-
-* [Microsoft SQL Server Developer Edition images for Windows Containers on DockerHub](https://hub.docker.com/r/microsoft/mssql-server-windows-developer/)
-
-#### Azure DevOps Services agent
-
-* [Azure DevOps Services agent images on DockerHub](https://hub.docker.com/r/microsoft/vsts-agent/)
-
-* [Azure DevOps Services agent images on GitHub](https://github.com/Microsoft/vsts-agent-docker)
-
-#### Operations Management Suite (OMS) Linux agent
-
-* [Operations Management Suite (OMS) Linux agent overview](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/docs/Docker-Instructions.md)
+The [relationship between Docker containers, images, and registries](../../standard/microservices-architecture/container-docker-introduction/docker-containers-images-registries.md) is an important concept when [architecting and building containerized applications or microservices](../../standard/microservices-architecture/architect-microservice-container-applications/index.md). This approach greatly shortens the time between development and deployment.
 
-* [Operations Management Suite (OMS) images on DockerHub](https://hub.docker.com/r/microsoft/oms/)
+### Dockerfile
 
-* [Operations Management Suite (OMS) images on GitHub](https://github.com/Microsoft/OMS-docker)
+A **Dockerfile** is a file that defines a set of instructions that creates an image. Each instruction in the **Dockerfile** creates a layer in the image. When you rebuild the image, only the layers that have changed are rebuilt. The **Dockerfile** can be distributed to others and allows them to recreate the image on their own system.
```

Comment:
> When you rebuild the image, only the layers that have changed are rebuilt.

As a blanket statement this is not always correct.  This is really dependent on how you build the image.  To achieve this you must have the build cache from the previous build.